src/main/java/Ui.java

```diff
@@ -0,0 +1,105 @@
+/**
+ * Handles interaction with users.
+ */
+public class Ui {
+    /**
+     * Greets user when Duke bot is activated.
+     * @return Duke response message.
+     */
+    public String greet() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        return "Hello! I'm Duke" + "\nWhat can I do for you?";
+    }
+
+    /**
+     * Prints response message on user inputs.
+     * @param string Response message by Duke bot.
+     */
+    public void printLine(String string) {
+        System.out.println(string);
+    }
+
+    /**
+     * Draws border on top and bottom of the response message.
+     * @param string Response message by Duke bot.
+     * @return Response message with border.
+     */
+    public String drawBorder(String string) {
+        String line = "_________________________________________________________________";
+        return line + "\n" + string + "\n" + line;
+    }
+
+    /**
+     * Prints bye message to user.
+     * @return Duke response message for bye.
+     */
+    public String bye() {
```

Comment:
Maybe you can name this as "printBye"? I think it will be more consistent with the rest of the method names, and it also sounds slightly more like an action :)

Reply:
Ohh seems like I missed out this. Alright, I'll edit it! thanks!